scripts/eurostat/health_determinants/bmi/process.py

```diff
@@ -0,0 +1,189 @@
+    This Class has requried methods to generate Cleaned CSV,
+    MCF and TMCF Files.
+    """
+    _import_name = "bmi"
+
+    _mcf_template = ("Node: dcid:{sv}\n"
+                     "typeOf: dcs:StatisticalVariable\n"
+                     "populationType: dcs:Person\n"
+                     "statType: dcs:measuredValue\n"
+                     "measuredProperty: dcs:count"
+                     "{denominator}"
+                     "{healthbehavior}"
+                     "{education}"
+                     "{exercise}"
+                     "{residence}"
+                     "{activity}"
+                     "{duration}"
+                     "{citizenship}"
+                     "{gender}"
+                     "{incomequin}"
+                     "{countryofbirth}"
+                     "{lev_limit}"
+                     "{frequency}"
+                     "\n{sv_name}"
+                     "\n")
+
+    _sv_properties_template = {
+        "healthbehavior":
+            "\nhealthBehavior: dcs:{proprty_value}",
+        "gender":
+            "\ngender: dcs:{proprty_value}",
+        "exercise":
+            "\nexerciseType: dcs:{proprty_value}",
+        "education":
+            "\neducationalAttainment: dcs:{proprty_value}",
+        "incomequin":
+            "\nincome: [{proprty_value}]",
+        "residence":
+            "\nplaceOfResidenceClassification: dcs:{proprty_value}",
+        "lev_limit":
+            "\nglobalActivityLimitationindicator: dcs:{proprty_value}",
+        "activity":
+            "",
+        "activity_temp":
+            "\nphysicalActivityEffortLevel: dcs:{proprty_value}Level",
+        "frequency":
+            "\nactivityFrequency: dcs:{proprty_value}",
+        "duration":
+            "",
+        "duration_temp":
+            "{proprty_value}",
+        "countryofbirth":
+            "\nnativity: dcs:{proprty_value}",
+        "citizenship":
+            "\ncitizenship: dcs:{proprty_value}",
+    }
+
+    _sv_value_to_property_mapping = {
+        "PhysicalActivity": "healthbehavior",
+        "Male": "gender",
+        "Female": "gender",
+        "Aerobic": "exercise",
+        "MuscleStrengthening": "exercise",
+        "Walking": "exercise",
+        "Cycling": "exercise",
+        "Education": "education",
+        "Percentile": "incomequin",
+        "Urban": "residence",
+        "Rural": "residence",
+        "Limitation": "lev_limit",
+        "ModerateActivity": "activity_temp",
+        "HeavyActivity": "activity_temp",
+        "NoActivity": "activity_temp",
+        "AtLeast30MinutesPerDay": "frequency",
+        "Minutes": "duration_temp",
+        "ForeignBorn": "countryofbirth",
+        "Native": "countryofbirth",
+        "ForeignWithin": "citizenship",
+        "ForeignOutside": "citizenship",
+        "Citizen": "citizenship",
+        "weight": "healthbehavior",
+        "Normal": "healthbehavior",
+        "Obese": "healthbehavior",
+        "Obesity": "healthbehavior",
```

Comment:
Can this be moved to config.py so it can also be used in denominator_mcf_generation?